runtime/server/controller.go

```diff
@@ -61,13 +62,19 @@ func (s *Server) ListResources(ctx context.Context, req *runtimev1.ListResources
 		return strings.Compare(an.Name, bn.Name)
 	})
 
+	isAdmin := auth.GetClaims(ctx).SecurityClaims().Admin()
+	if isAdmin && req.SkipChecks {
+		return &runtimev1.ListResourcesResponse{Resources: rs}, nil
+	}
```

Comment:
If `skip_checks` is passed and the caller is not an admin, it should error instead of failing silently by applying checks anyway. For example:
```suggestion
	if req.SkipChecks {
		if !auth.GetClaims(ctx).SecurityClaims().Admin() {
			return nil, ErrForbidden
		}
		return &runtimev1.ListResourcesResponse{Resources: rs}, nil
	}
```